src/main/java/org/shanerx/tradeshop/bitrade/BiShopCreateEventListener.java

```diff
@@ -44,51 +45,50 @@ public BiShopCreateEventListener(TradeShop instance) {
 	}
 	
 	@SuppressWarnings("deprecation")
-	@EventHandler
+	@EventHandler(priority = EventPriority.HIGH)
 	public void onSignChange(SignChangeEvent event) throws InterruptedException {
-		//	BlockState state = event.getBlock().getState();
 		Player player = event.getPlayer();
 		Sign s = (Sign) event.getBlock().getState();
-		if (!(event.getLine(0).equalsIgnoreCase("[BiTrade]"))) {
+		
+		if (!event.getLine(0).equalsIgnoreCase("[BiTrade]")) {
```

Comment:
When I initially wrote the plugin (Idk how it looks like now) it automatically changed the case to [Trade]/[iTrade], so #equalsIgnoreCase was superficial. I am not sure how it works now.